dbt-snowflake/src/dbt/include/snowflake/macros/relations/table/create.sql

```diff
@@ -1,69 +1,29 @@
 {% macro snowflake__create_table_as(temporary, relation, compiled_code, language='sql') -%}
 
-  {%- if relation.is_iceberg_format and not adapter.behavior.enable_iceberg_materializations.no_warn %}
-    {% do exceptions.raise_compiler_error('Was unable to create model as Iceberg Table Format. Please set the `enable_iceberg_materializations` behavior flag to True in your dbt_project.yml. For more information, go to https://docs.getdbt.com/reference/resource-configs/snowflake-configs#iceberg-table-format') %}
-  {%- endif %}
-
-  {%- set materialization_prefix = relation.get_ddl_prefix_for_create(config.model.config, temporary) -%}
-  {%- set alter_prefix = relation.get_ddl_prefix_for_alter() -%}
-
-  {# Generate DDL/DML #}
-  {%- if language == 'sql' -%}
```

Comment:
Move the contents of the passing `if` condition to `relations/table/create/standard.sql`.